common/src/main/scala/org/apache/celeborn/common/network/CelebornRackResolver.scala

```diff
@@ -46,8 +48,8 @@ class CelebornRackResolver(celebornConf: CelebornConf) extends Logging {
     }
   }
 
-  def resolve(hostName: String): String = {
-    coreResolve(Seq(hostName)).head.getNetworkLocation
+  def resolve(hostName: String): Node = {
```

Comment:
It's better to cache the hostName -> Node in this class to avoid resolve every time.

Reply:
> It's better to cache the hostName -> Node in this class to avoid resolve every time.

The underline implement is `CachedDNSToSwitchMapping`, so we don't need to do this here.